knowledge-base/do-i-need-to-use-a-data-source-component.md

```diff
@@ -0,0 +1,30 @@
+---
+title: Do I need to use a Data Source component?
+description: Do I need to use a Data Source component?.
+type: how-to
+page_title: Do I need to use a Data Source component?
+slug: do-i-need-to-use-a-data-source-component
+res_type: kb
+---
+
+## Description
+ 
+Do I need to use a [Data Source component](../connecting-to-data-data-source-components)?  
```

Comment:
Add more details. In what scenarios the questions is asked? Bring more infor about the data source component and how and why it is helpful to have it or not.